R/ResampleInstance.R

```diff
@@ -61,7 +61,26 @@ makeResampleInstance = function(desc, task, size, ...) {
   if (length(blocking) && desc$stratify)
     stop("Blocking can currently not be mixed with stratification in resampling!")
 
-  if (length(blocking)) {
+  # 'fixed' only exists by default for 'CV' -> is.null(desc$grouping)
+  # only use this way of blocking of 'fixed = FALSE' -> is.null(desc$grouping)
+  if(is.null(desc$fixed)) {
```

Comment:
Space after `if` (isn't this supposed to be checked automatically?).

Reply:
Also I would give `fixed` a default value in `makeResampleDesc` so that this can never be `NULL` to remove the additional check.

Reply:
Set a default value.